packages/integration-tests/src/package_manager_sanity_checks.test.ts

```diff
@@ -0,0 +1,124 @@
+import { execa } from 'execa';
+import * as fsp from 'fs/promises';
+import * as path from 'path';
+import * as os from 'os';
+import { after, afterEach, before, beforeEach, describe, it } from 'node:test';
+import assert from 'assert';
+import { glob } from 'glob';
+import { BackendIdentifier } from '@aws-amplify/plugin-types';
+import {
+  CloudFormationClient,
+  DeleteStackCommand,
+} from '@aws-sdk/client-cloudformation';
+import { BackendIdentifierConversions } from '@aws-amplify/platform-core';
+import { getClientConfigPath } from '@aws-amplify/client-config';
+import { TestBranch, amplifyAppPool } from './amplify_app_pool.js';
+import { e2eToolingClientConfig } from './e2e_tooling_client_config.js';
+import {
+  type PackageManager,
+  type PackageManagerExecutable,
+  setupPackageManager,
+} from './setup_package_manager.js';
+import {
+  runPackageManager,
+  runWithPackageManager,
+} from './process-controller/process_controller.js';
+
+/**
+ * TODO: This test will be refactored to use the test from health-checks.test.ts and run with different package managers.
```

Comment:
I'd remove the TODO for now. we have work logged in gh issues.

Reply:
Done